File: Plugin/plugin-monitor/src/main/java/com/kronos/plugin/monitor/repo/data/GitStatus.java
package com.kronos.plugin.monitor.repo.data;

public class GitStatus {

    public String path;
    public String commit;
    public String branch;
    public String status;

    @Override
    public String toString() {
        return "GitStatus{" +
                "path='" + path + '\'' +
                ", commit='" + commit + '\'' +
                ", branch='" + branch + '\'' +
                ", status='" + status + '\'' +
                '}';
    }

    public String toDec() {
        return "path=" + path + "\n" +
                "branch=" + branch + "\n" +
                "commit=" + commit + "\n" +
                "status=" + status + "\n";
    }
}

File: Plugin/plugin-monitor/src/main/java/com/kronos/plugin/monitor/repo/data/GitStatus.kt
package com.kronos.plugin.monitor.repo.data

class GitStatus {

    var path: String? = null
    var commit: String? = null
    var branch: String? = null
    var status: String? = null

    override fun toString(): String {
        return "GitStatus{" +
                "path='" + path + '\'' +
                ", commit='" + commit + '\'' +
                ", branch='" + branch + '\'' +
                ", status='" + status + '\'' +
                '}'
    }

    fun toDec(): String {
        return """
               path=$path
               branch=$branch
               commit=$commit
               status=$status
               
               """.trimIndent()
    }
}